test/servinge2e/tracing_test.go

```diff
@@ -0,0 +1,263 @@
+package servinge2e
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	jaegermodel "github.com/jaegertracing/jaeger/model"
+	jaegerapi "github.com/jaegertracing/jaeger/proto-gen/api_v2"
+	"github.com/openshift-knative/serverless-operator/test"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/pkg/test/spoof"
+)
+
+const (
+	testNamespace    = "serverless-tests"
+	requestCount     = 100
+	tracingNamespace = "istio-system"
+	jaegerName       = "jaeger"
```

Comment:
```suggestion
	jaegerName       = "jaeger"
	jaegerQueryPort  = 16685
```

Let's give a name to `16685` as it is difficult to remember what is this number :sweat_smile: